src/m365/tenant/commands/serviceannouncement/serviceannouncement-healthissue-get.ts

```diff
@@ -0,0 +1,64 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  issueId: string;
+}
+
+class TenantServiceAnnouncementHealthissueGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.SERVICEANNOUNCEMENT_HEALTHISSUE_GET;
+  }
+
+  public get description(): string {
+    return 'Gets a specified service health issue for tenant';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'title'];
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.issueId = typeof args.options.issueId !== 'undefined';
```

Comment:
Because `issueId` is required, there's no point in tracking its usage because it would always be true